numpy/distutils/command/build_ext.py

```diff
@@ -281,7 +281,7 @@ def run(self):
                 runtime_lib = os.path.join(self.extra_dll_dir, fn)
                 copy_file(runtime_lib, shared_lib_dir)
 
-    def swig_sources(self, sources):
+    def swig_sources(self, *args):
```

Comment:
the function signature is `swig_sources(self, sources, extensions)`, this fix is forward-compatible as well.

Reply:
This doesn't work, because the `return sources` on line 286 now gives a `NameError`

Reply:
fixed

Reply:
Why not `sources, extensions=None`?

Reply:
adopted